src/syscheckd/src/create_db.c

```diff
@@ -1660,26 +1660,6 @@ void free_file_data(fim_file_data * data) {
     os_free(data);
 }
 
-
-void free_entry(fim_entry * entry) {
```

Comment:
We could review why this function is no longer used in the FIM scan. I don't find a replacement to free a FIM entry.